examples/serviceaccount/main.tf

```
@@ -0,0 +1,17 @@
+terraform {
+  required_providers {
+    minio = {
+      source = "aminueza/minio"
+      version = "= 1.6.0"
```

In a follow-up PR, we could change the following, to make it work with any future version :)
```suggestion
      version = ">= 1.0.0"
```

---

Updated in https://github.com/aminueza/terraform-provider-minio/pull/361.